Assets/MRTK/Examples/Demos/Input/Scenes/InputActions/InputActions.MixedRealityGesturesProfile.asset

```diff
@@ -37,3 +37,9 @@ MonoBehaviour:
       id: 9
       description: Manipulate Action
       axisConstraint: 0
+  - description: Select
```

Comment:
Can you explain what this new field is doing?

It would also be good to add an explanation of this new input field in [the gestures documentation](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Input/Gestures.html), as part of this PR.

Reply:
This change ended up being extraneous, discarding